label_options/usaid-laos-ev-label-options.json

```diff
@@ -1,124 +1,123 @@
-        "e-bike": "E-bike",
-        "bike": "Bicycle",
-        "drove_alone": "Car Drove Alone",
-        "shared_ride": "Car Shared Ride",
-        "e_car_drove_alone": "E-Car Drove Alone",
-        "e_car_shared_ride": "E-Car Shared Ride",
-        "taxi": "Taxi/Loca/inDrive",
-        "bus": "Bus",
-        "train": "Train",
-        "free_shuttle": "Free Shuttle",
-        "air": "Airplane",
-        "not_a_trip": "Not a trip",
-        "home": "Home",
-        "work": "To Work",
-        "at_work": "At Work",
-        "school": "School",
-        "transit_transfer": "Transit transfer",
-        "shopping": "Shopping",
-        "meal": "Meal",
-        "pick_drop_person": "Pick-up/ Drop off Person",
-        "pick_drop_item": "Pick-up/ Drop off Item",
-        "personal_med": "Personal/ Medical",
-        "access_recreation": "Access Recreation",
-        "exercise": "Recreation/ Exercise",
-        "entertainment": "Entertainment/ Social",
-        "religious": "Religious",
-        "other": "Other"
-      },
-      "lo": {
-        "walk": "ດ້ວຍການຍ່າງ",
-        "e-auto_rickshaw":"ລົດ 3 ລໍ້ໄຟຟ້າ ຫລື ຕຸກຕຸກໄຟຟ້າ",
-        "auto_rickshaw":"ເດີນທາດ້ວຍ ລົດຕຸກຕຸກ ຫລື ລົດສາມລໍ້",
-        "motorcycle":"ລົດຈັກ",
-        "e-bike": "ວຍລົດຈັກໄຟຟ້າ",
-        "bike": "ລົດຖີບ",
-        "drove_alone": "ເດີນທາງ ດ້ວຍລົດໃຫ່ຍ ເຊີ່ງເປັນລົດທີ່ຂັບເອງ",
-        "shared_ride": "ເດີນທາງດ້ວຍລົດໃຫ່ຍ ຮ່ວມກັບລົດຄົນອຶ່ນ",
-        "e_car_drove_alone": "ດ້ວຍການຂັບລົດໄຟຟ້າໄປເອງ",
-        "e_car_shared_ride": "ດ້ວຍການຈ້າງລົດໄຟຟ້າໄປ",
-        "taxi": "ແທັກຊີ",
-        "bus": "ລົດເມ",
-        "train": "ລົດໄຟ",
-        "free_shuttle": "ລົດຮັບສົ່ງຟຣີ",
-        "air": "ຍົນ",
-        "not_a_trip": "ບໍ່ແມ່ນການເດີນທາງ",
-        "home": "ບ້ານ",
-        "work": "ໄປເຮັດວຽກ",
-        "at_work": "ຢູ່ບ່ອນເຮັດວຽກ",
-        "school": "ໄປໂຮງຮຽນ",
-        "transit_transfer": "ການຖ່າຍໂອນການເດີນທາງ",
-        "shopping": "ຊອບປິ້ງ",
-        "meal": "ອາຫານ",
-        "pick_drop_person": "ໄປຮັບ ຫລື ສົນ ຄົນ",
-        "pick_drop_item": "ໄປຮັບ ຫລື ສົ່ງສິນຄ້າ",
-        "personal_med": "ໄປຫາໝໍ",
-        "access_recreation": "ເຂົ້າເຖິງການພັກຜ່ອນ",
-        "exercise": "ພັກຜ່ອນ/ອອກກຳລັງກາຍ",
-        "entertainment": "ບັນເທີງ/ສັງຄົມ",
-        "religious": "ຈຸດປະສົງທາງສາດສະໜາ",
-        "other": "ອື່ນໆ"
-      }
+  "MODE": [
+    {"value":"walk", "base_mode": "WALKING",
+      "baseMode":"WALKING", "met_equivalent":"WALKING", "kgCo2PerKm": 0},
+    {"value":"e-auto_rickshaw", "base_mode": "MOPED", "icon": "rickshaw-electric", "footprint": { "electric": { "wh_per_km": "TODO" }},
+      "baseMode":"MOPED", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.085416859},
+    {"value":"auto_rickshaw", "base_mode": "MOPED", "icon": "rickshaw", "footprint": { "electric": { "wh_per_km": "TODO" }},
+      "baseMode":"MOPED", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.231943784},
+    {"value":"motorcycle", "base_mode": "MOPED", "footprint": { "gasoline": { "wh_per_km": "TODO" }},
```

Comment:
TODO need updated `footprint`s for `e-auto_rickshaw`, `auto_rickshaw`, and `motorcycle`.
@Abby-Wheelis do you know where the `kgCo2PerKm` values came from?

Reply:
I never documented where I got those from ... unfortunately, looking and documenting now:

e-rickshaw:  341.82 Wh/km [from a study in Bangkok](https://saemobilus.sae.org/papers/dynamic-modeling-electric-tuk-tuk-predicting-energy-consumption-bangkok-driving-condition-2013-01-0113)

auto_rickshaw:  (gasoline) 10km/L -> 970 Wh/km
motorcycle: (gasoline) 20.5km/L -> 473.17 Wh/km 

based on [an energy inventory](https://www.eria.org/uploads/media/0_Lao_PDR_Energy_Statistics_2018_complete_book.pdf), and assumed 9700 wh/L